hooks/environment

```diff
@@ -12,6 +12,22 @@ check_command() {
   fi
 }
 
+# the buildkite agent redactor has been added from version 3.66.0 and on. So we need to check it.
+# https://github.com/buildkite/agent/releases/tag/v3.66.0
+check_buildkite_version() {
+  local version_output
+  version_output=$(buildkite-agent --version)
+  local version
+  version=$(echo "$version_output" | grep -oP '\d+\.\d+\.\d+')
```

Comment:
Does this need to be wrapped as a string:
```suggestion
  version="$(echo "$version_output" | grep -oP '\d+\.\d+\.\d+')"
```

Or is it intentionally as a number? (if so, maybe a comment?)

Reply:
The version output has quotes around it. Also we grep the semantic version so it is not a number.

Reply:
Correct, but "to be safe", should we then also make sure that we're wrapping the resultant value as a non-breaking string?

Reply:
changed in e7d7634ec2ebe380068105241a9464198b0d4194